action.yml

```diff
@@ -45,6 +45,20 @@ inputs:
   version:
     description: 'The runtime on which the app will run.'
     required: false
+  builder:
+    description: '(Enterprise Tier Only) Build service builder used to build the executable.'
+    required: false
+  build-cpu:
+    description: '(Enterprise Tier Only) CPU resource quantity. Should be 500m or number of CPU cores. Default: 1'
```

Comment:
Maybe

CPU resource quantity for build container.

Same suffix for the memory.